### PR TITLE
Minor Changes

### PR DESCRIPTION
**Pull request by joinimran**
As suggested by the user, I made a few changes to reflect the document more clear and easy to understand. 

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/7965

### CODE REVIEW

**Comment by joinimran** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/client-management/connect-to-remote-aadj-pc.md [hunk 1]
```diff
@@ -22,13 +22,10 @@ ms.topic: article
 
 - Windows 10
 
-From its release, Windows 10 has supported remote connections to PCs joined to Active Directory. Starting in Windows 10, version 1607, you can also connect to a remote PC that is [joined to Azure Active Directory (Azure AD)](https://docs.microsoft.com/azure/active-directory/user-help/device-management-azuread-joined-devices-setup).
+From its release, Windows 10 has supported remote connections to PCs joined to Active Directory. Starting in Windows 10, version 1607, you can also connect to a remote PC that is [joined to Azure Active Directory (Azure AD)](https://docs.microsoft.com/azure/active-directory/user-help/device-management-azuread-joined-devices-setup). Starting in Windows 10, version 1809, you can [use biometrics to authenticate to a remote desktop session.](https://docs.microsoft.com/windows/whats-new/whats-new-windows-10-version-1809#remote-desktop-with-biometrics)
```
Comment (JohanFreelancer9) [inline windows/client-management/connect-to-remote-aadj-pc.md:25]:
```suggestion
From its release, Windows 10 has supported remote connections to PCs joined to Active Directory. Starting in Windows 10, version 1607, you can also connect to a remote PC that is [joined to Azure Active Directory (Azure AD)](https://docs.microsoft.com/azure/active-directory/user-help/device-management-azuread-joined-devices-setup). Starting in Windows 10, version 1809, you can [use biometrics to authenticate to a remote desktop session](https://docs.microsoft.com/windows/whats-new/whats-new-windows-10-version-1809#remote-desktop-with-biometrics).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Comment by joinimran** [comment]
Thanks @JohanFreelancer9 - Ready to go live @e0i

**Review by Yoshio123 — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**